util/src/main/kotlin/ProcessCapture.kt

```diff
@@ -39,10 +39,10 @@ class ProcessCapture(workingDir: File?, vararg command: String) {
     val commandLine = command.joinToString(" ")
 
     @Suppress("UnsafeCallOnNullableType")
-    val stdoutFile = File.createTempFile(command.first(), ".stdout")!!
+    val stdoutFile = File.createTempFile(command.first().padEnd(3, '_'), ".stdout")!!
```

Comment:
How about extracting this common code to a private val now:

    private val tempPrefix = command.first().padEnd(3, '_')